pkg/recipes/configloader/environment.go

```diff
@@ -161,6 +161,10 @@ func getRecipeDefinition(environment *v20231001preview.EnvironmentResource, reci
 	switch c := found.(type) {
 	case *v20231001preview.TerraformRecipeProperties:
 		definition.TemplateVersion = *c.TemplateVersion
+	case *v20231001preview.BicepRecipeProperties:
+		if c.PlainHTTP != nil {
```

Comment:
Doesn't this apply to Terraform registries also?

Reply:
No, its not needed for terraform registry. Since we only use registery to publish bicep recipes today.